### PR TITLE
Update ASR_with_SpeakerDiarization.ipynb tutorial

### PR DESCRIPTION
**Pull request by tango4j**
Updating ASR_with_SpeakerDiarization.ipynb.
Currently, ASR with diarization only supports QuartzNet. We will be adding more models later on.

### CODE REVIEW

File: tutorials/speaker_tasks/ASR_with_SpeakerDiarization.ipynb [hunk 1]
```diff
@@ -485,42 +447,31 @@
...
-    "        print()\n",
-    "        idx+=1\n",
-    "        start_point,end_point,speaker=labels[idx].split()\n",
-    "        print(\"{} [{:.2f} - {:.2f} sec]\".format(speaker,float(start_point),float(end_point)),end=\" \")\n",
-    "        print(words[j],end=\" \")\n",
-    "    pos_prev = pos_end\n",
-    "\n",
-    "\n",
-    "print(words[j+1],end=\" \")"
+    "transcription_path_to_file = f\"{ROOT}/json_result/an4_diarize_test.json\"\n",
```
Comment (nithinraok) [inline tutorials/speaker_tasks/ASR_with_SpeakerDiarization.ipynb:471]: path should be `{ROOT}/asr_with_diar/json_result/an4_diarize_test.json\`

File: tutorials/speaker_tasks/ASR_with_SpeakerDiarization.ipynb [hunk 2]
```diff
@@ -485,42 +447,31 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "color=get_color(signal,labels)\n",
-    "show_figure(signal,'audio with speaker labels',color)\n",
-    "display(Audio(signal,rate=16000))"
+    "transcription_path_to_file = f\"{ROOT}/asr_with_diartranscript_with_speaker_labels/an4_diarize_test.txt\"\n",
```
Comment (nithinraok) [inline tutorials/speaker_tasks/ASR_with_SpeakerDiarization.ipynb:450]: `/` is missing after asr_with_diar folder. 
Correct path
 `{ROOT}/asr_with_diar/transcript_with_speaker_labels/an4_diarize_test.txt`

**Review by nithinraok — CHANGES_REQUESTED** [review]
Minor fixes. look very good to me otherwise

File: tutorials/speaker_tasks/ASR_with_SpeakerDiarization.ipynb (hunk 2 above)
Comment (fayejf) [reply tutorials/speaker_tasks/ASR_with_SpeakerDiarization.ipynb]: fixed

File: tutorials/speaker_tasks/ASR_with_SpeakerDiarization.ipynb (hunk 1 above)
Comment (fayejf) [reply tutorials/speaker_tasks/ASR_with_SpeakerDiarization.ipynb]: fixed

**Review by fayejf — APPROVED** [review]
LGTM! I've added some minor typo and fixes, feel free to bring back.

**Review by nithinraok — APPROVED** [review]

**Pull request merged**